tests/search/grouping/grouping.rb

```diff
@@ -7,8 +7,7 @@ class Grouping < IndexedSearchTest
 
   def setup
     set_owner("bjorncs")
-    deploy_app(SearchApp.new.sd("#{selfdir}/purchase.sd").
-               feeder_options(FeederOptions.new.max_pending_docs(1)))
+    deploy_app(SearchApp.new.sd("#{selfdir}/purchase.sd"))
```

Comment:
I wonder if max_pending=1 is required to get a deterministic test.
Is there another way to achieve that, or does this mean that it has not been working ?

Reply:
This config has not been working but there is config for the feeder in the test that will do what is wanted I think:

`    feed_and_wait_for_docs("purchase", 20, :file => "#{selfdir}/docs.xml", :maxpending => 1);`

Reply:
Perfect, just leftovers then.